fcrepo-jms/src/main/java/org/fcrepo/jms/observer/JMSTopicPublisher.java

```diff
@@ -84,6 +85,20 @@ public void publishJCREvent(final FedoraEvent fedoraEvent) throws JMSException,
     }
 
     /**
+     * Subscribe to special fedora event
+     * @param fixityEvent
+     */
+    @Subscribe
+    public void publishFedoraEvent(final FixityEvent fixityEvent) throws JMSException,
```

Comment:
That is completely beside my point here. In any event, see above for that objection.